x/feegrant/module/module.go

```diff
@@ -13,13 +13,17 @@ import (
 	appmodulev2 "cosmossdk.io/core/appmodule/v2"
 	"cosmossdk.io/core/transaction"
 	"cosmossdk.io/errors"
+	"cosmossdk.io/runtime/v2"
+	"cosmossdk.io/x/auth/ante"
```

Comment:
isn't the idea to remove the dependency on x/auth and write a fresh TxValidator?

Reply:
I believe so. It is no longer present, we are moving this decorator to x/auth as discussed in the server sync call